deploy/README.md

```diff
@@ -1129,3 +1129,246 @@ Ensure that no keys duplicated across `gramex.yaml` files and restart gramex.
 - `cannot set up log`:  The `log:` section has an error. It may not be a dict,
   or refer to a file / folder that cannot be accessed, etc.
 - `Gramex 1.xx.x is available`: A newer version of Gramex is available. Upgrade
+
+## Good security practices
+
+A curated list of security practices for real-life application deployments is presented here.
+
+### Session timeout
+
+Applications may need to timeout user sessions periodically. We can configure this as below:
+
+```yaml
+# gramex.yaml
+application/login:
+  pattern: /$YAMLURL/login
+  handler: DBAuth
+  kwargs:
+    session_expiry: 0.0207           # session expires after 30 mins
+    session_inactive: 0.0207         # session expires after 30 mins of inactivity
+```
+
+### Disable directory listing
+
+Directory listing isn't a recommended practice as it reveals exact file names. One can disable directory listing at a root level as below:
+
+```yaml
+# gramex.yaml
+handlers:
+  FileHandler:
+    ignore:                             # accepts a list of files to be ignored
+      - '*.yaml'                        # YAML defines routes, user credentials
+      - '*.git*'                        # git files
+      - '*.json'
+      - '*Dockerfile'
+      - '*nohup.out'                    # logs
+      - '*.git/*'
+      - '*ui/*'
```

Comment:
Would it be useful to mention the `allow` kwarg here as well? I might find it easier to just `allow` three types of files, instead of ignoring seven of them.

Reply:
although this is only to demonstrate file ignore option will add allow option too.